massa-execution-worker/src/interface_impl.rs

```diff
@@ -581,7 +581,7 @@ impl Interface for InterfaceImpl {
             validity_start: Slot::new(validity_start.0, validity_start.1),
             validity_end: Slot::new(validity_end.0, validity_end.1),
             max_gas,
-            gas_price: Amount::from_raw(gas_price),
+            fee: Amount::from_raw(raw_fee),
```

Comment:
And I reckon that the amount burnt from the message sender must also take into consideration the duration of the message validity, which is not the case either since nothing is burnt except coins

Reply:
Thanks! Did you have a chance to investigate? Any findings?

Reply:
I think it's a good idea to open an issue, and address this in a follow-up PR.

Reply:
Ah thanks @AurelienFT, I see you've just opened https://github.com/massalabs/massa/issues/3234